src/common/configuration.py

```
@@ -0,0 +1,32 @@
+    provider: str | None = None
+
+    # extended config
+    memory: MemoryConfiguration = field(default_factory=MemoryConfiguration)
+
+    @property
+    def langgraph_configurables(self) -> dict[str, Any]:
+        """
+        Returns a dictionary of configuration fields excluding those specified in the internal exclusion list.
+        
+        The returned dictionary contains the instance's attributes except for those whose keys are present in the `_LANGGRAPH_CONFIGURABLES` list.
+        """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)